File: CStoreManagement/app/src/main/java/com/c_store/zhiyazhang/cstoremanagement/view/TestActivity.java
package com.c_store.zhiyazhang.cstoremanagement.view;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.support.annotation.Nullable;

import com.c_store.zhiyazhang.cstoremanagement.R;
import com.c_store.zhiyazhang.cstoremanagement.utils.activity.MyActivity;

/**
 * Created by zhiya.zhang
 * on 2017/6/9 16:23.
 */

@SuppressLint("Registered")
public class TestActivity extends MyActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    protected int getLayoutId() {
        return R.layout.test;
    }
}

File: MyKotlinApplication/app/src/main/java/com/zhiyazhang/mykotlinapplication/view/activity/TestActivity.kt
package com.zhiyazhang.mykotlinapplication.view.activity

import android.os.Bundle
import com.zhiyazhang.mykotlinapplication.R
import com.zhiyazhang.mykotlinapplication.utils.MyActivity

/**
 * Created by zhiya.zhang
 * on 2017/6/2 16:39.
 */
class TestActivity : MyActivity() {
    var mList = arrayListOf("axx", "bxx", "cxx")

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

    }

    override val layoutId: Int get() = R.layout.activity_signin

}

